tests/integration_tests/test_cli.py

```diff
@@ -196,6 +198,7 @@ def test_export_savedmodel_cli(csv_filename):
 
 
 @pytest.mark.distributed
+@pytest.mark.skipIf(LooseVersion(platform.python_version()) >= LooseVersion('3.8'))
```

Comment:
3.8 should be fine right? Just need to skip 3.9.

Reply:
Done.